src/app/plugins/quote.plugin.ts

```diff
@@ -47,15 +61,16 @@ export default class QuotePlugin extends Plugin {
       return true;
     }
 
-    return args[0] === 'list' || this._VALID_TAGS.includes(args[0].toLowerCase());
+    const [tagInput] = args;
+    return tagInput === 'list' || QuotePlugin._VALID_TAGS.includes(tagInput.toLowerCase());

```

Comment:
I should've caught this in the first PR for this plugin,
but this should be lowercased
```suggestion
    return tagInput.toLowerCase() === 'list' || QuotePlugin._VALID_TAGS.includes(tagInput.toLowerCase());

```